### PR TITLE
general.css Video Skin Class Stlye Improvement

### PR DESCRIPTION
**Pull request by Smurphonomics**
Previously, the global video element was styled to make it look better on the stand alone page. However; the class `.showasset-light` is the video skin class assigned to the stand alone page only.

Based on this, I have optimised the style on this class and removed parts no longer needed from the `video-player` class, thereby not affecting any other pages for areas only required on the stand alone page.

### CODE REVIEW

File: public/assets/css/general.css [hunk 1]
```diff
@@ -516,18 +516,23 @@ table {
...
-    padding:6px;
 }
 
 .showlite-asset {
     max-width: 50%;
+    margin: 0 auto;
+    margin-top: 2%;
+    padding: 6px;
+    border: 1px solid #a2a2a2;
+//Optional Box Shadow - Looks better on a darker than white background.
```
Comment (billbitt) [inline public/assets/css/general.css]: I'm not sure about the box shadow in general, but I do think it works better on `#video-player` than on regular `showlite-asset` assets (e.g. pictures).  How about moving it to #video-player.

File: public/assets/css/general.css [hunk 2]
```diff
@@ -516,18 +516,23 @@ table {
...
-    border: 1px solid #d0d0d0;
-    padding:6px;
 }
 
 .showlite-asset {
     max-width: 50%;
+    margin: 0 auto;
+    margin-top: 2%;
+    padding: 6px;
+    border: 1px solid #a2a2a2;
```
Comment (billbitt) [inline public/assets/css/general.css]: I think I prefer the images without a border., but also the `margin: 0 auto;` doesn't seem to be taking affect.  I'd like to see what it looks like if the images are centered like the video.

**Comment by Smurphonomics** [comment]
Yes. The above CSS was for video only. Does .showasset-light affect images too?

All images need are positions, such as `center`, `center `and maybe scaling to fit in the page?

I really need to set this app up so I can test locally. Will do this today!

**Comment by Smurphonomics** [comment]
ID's need to be applied. What are the ID's for the following 2 pages?

https://spee.ch/b39388bb9ffa8946dde2acd49fbdc01561eb2edd/Speechless3
https://spee.ch/b/Speechless3.mp4

**Comment by billbitt** [comment]
https://spee.ch/b39388bb9ffa8946dde2acd49fbdc01561eb2edd/Speechless3 - `<body id="show-body">`
https://spee.ch/b/Speechless3.mp4 - `<body id="showlite-body">`

**Pull request merged**